### PR TITLE
Changing prefix from b to u in migration files to fix type mismatch

### PR DESCRIPTION
**Pull request by jinder1s**
The 'b' prefix before strings in migration files causes issues in python3. This replaces 'b' prefix with 'u' in migration files and adds 'u' prefix in relevant places in model.py
- this should not cause any issues in python 2

PR also moves back the pin on djangorestframework back to 3.7.7

### CODE REVIEW

File: requirements/dev.txt [hunk 1]
```diff
@@ -41,66 +40,59 @@ edx-opaque-keys==2.0.0    # via edx-drf-extensions
 enum34==1.1.6
 filelock==3.0.12          # via tox
 fs==2.4.11
-funcsigs==1.0.2           # via mock
```
Comment (jmbowman) [inline requirements/dev.txt]: Some of these removed dependencies still seem to be needed; did you run `make upgrade` under Python 2.7, and is the `--rebuild` option being passed to pip-compile?

File: requirements/dev.txt [hunk 2]
```diff
@@ -41,66 +40,59 @@ edx-opaque-keys==2.0.0    # via edx-drf-extensions
...
+six==1.12.0               # via astroid, diff-cover, edx-drf-extensions, edx-lint, edx-opaque-keys, fs, mock, packaging, pip-tools, pyjwkest, python-dateutil, stevedore, tox
 snowballstemmer==2.0.0    # via pydocstyle
 stevedore==1.31.0         # via edx-opaque-keys
 toml==0.10.0              # via tox
 tox-battery==0.5.1
 tox==3.14.0
+typed-ast==1.4.0          # via astroid
 typing==3.7.4.1           # via fs
-urllib3[secure]==1.25.6   # via coveralls, requests
+urllib3==1.25.6           # via requests
```
Comment (jmbowman) [inline requirements/dev.txt]: I don't think we want to drop the `[secure]` clarification, that brings in some extra dependencies.

File: requirements/base.in [hunk 3]
```diff
@@ -4,7 +4,7 @@ boto
 Django>=1.11,<2
 django-model-utils
 edx-django-oauth2-provider
-djangorestframework==3.9.4
+djangorestframework==3.7.7
```
[the remark this answers is not in the document]
Comment (azarembok) [reply requirements/base.in]: I was the one who attempted this upgrade (not realizing that I was creating a conflict with edx-platform). Jinder and I discusssed and I agree with changing it back until we can figure out how to resolve the conflict.

File: requirements/dev.txt [hunk 4]
```diff
@@ -4,34 +4,33 @@
 #
 #    make upgrade
 #
+--index-url http://edx.devstack.devpi:3141/root/pypi/+simple/
+--extra-index-url https://pypi.python.org/simple
+--trusted-host edx.devstack.devpi
```
[the remark this answers is not in the document]
Comment (jinder1s) [reply requirements/dev.txt]: I was running make upgrade in edx-platform, I reran things and fixed this

File: requirements/dev.txt (hunk 1 above)
Comment (jinder1s) [reply requirements/dev.txt]: I was running make upgrade in edx-platform, I reran things and fixed this

File: requirements/dev.txt (hunk 4 above)
[the remark this answers is not in the document]
Comment (jinder1s) [reply requirements/dev.txt]: I was running make upgrade in edx-platform, I reran things and fixed this

**Pull request merged**